pkg/config/constants.go

```diff
@@ -8,6 +8,8 @@ const (
 
 	RoleDefault = "default"
 	RoleWitness = "witness"
+	RoleMgmt    = "mgmt"
```

Comment:
how about `management` to be consistent with the label `HarvesterMgmtNodeLabelKey`.

Reply:
updated!